code/game/objects/machinery/doors/airlock_types.dm

```diff
@@ -543,15 +543,15 @@
 	req_access = list(ACCESS_MARINE_RESEARCH)
 
 /obj/machinery/door/airlock/mainship/medical/rebel/glass/research
-	req_access = list(ACCESS_MARINE_RESEARCH_REBEL)
+	req_access = list(ACCESS_MARINE_RESEARCH)
 
 /obj/machinery/door/airlock/mainship/research
 	name = "\improper Research Airlock"
 	icon = 'icons/obj/doors/mainship/medidoor.dmi'
 	req_access = list(ACCESS_MARINE_RESEARCH)
 
 /obj/machinery/door/airlock/mainship/research
-	req_access = list(ACCESS_MARINE_RESEARCH_REBEL)
+	req_access = list(ACCESS_MARINE_RESEARCH)
```

Comment:
This'll fix the issue now, but it's still defining `/obj/machinery/door/airlock/mainship/research` twice. (Once on L548, once on L553)
Better way to do it would be to either just remove the second definition entirely, or to add `/rebel` on the end to make it a new subtype:
```dm
/obj/machinery/door/airlock/mainship/research/rebel
	req_access = list(ACCESS_MARINE_RESEARCH_REBEL)
```